fydetabduo-post-install/fydetab.install

```diff
@@ -2,6 +2,8 @@ post_install() {
     systemctl enable --now fydetabduo.service
     systemctl enable --now bluetooth-fydetab.service
     systemctl enable --now touchegg.service
+
+    dconf write /org/cinnamon/settings-daemon/peripherals/touchscreen/orientation-lock false
```

Comment:
If you run `dconf` here, it runs as root. I don't think that would update the default user's configuration.